src/components/ArcGISWebMap.tsx

```diff
@@ -159,14 +169,138 @@ function FormElementArcGISWebMap({
     }
   }, [element, onChange, value])
 
+  const getGeometryPoints = (
+    geom: __esri.Geometry,
+    spatialRef: SpatialReference,
+  ) => {
+    if (geom.type === 'polygon') {
+      const polygon = geom as Polygon
+      return polygon.rings[0].map(
```

Comment:
more of a question than a 'comment', but why is the geometry points  only ever calculated on `ring[0]` or `path[0]`?